ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -70,3 +70,38 @@ def test_torch_cat(
         dim=unique_idx,
         out=None,
     )
+
+    
+# reshape
+@given(
+    xs_n_input_dtypes_n_unique_idx=_arrays_idx_n_dtypes(),
+    as_variable=helpers.array_bools(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.torch.reshape"
+    ),
+    native_array=helpers.array_bools(),
+    with_out=st.booleans(),
+)
+def test_torch_reshape(
+    xs_n_input_dtypes_n_unique_idx,
+    as_variable,
+    num_positional_args,
+    native_array,
+    with_out,
+    fw,
+):
+    xs, input_dtypes, unique_idx = xs_n_input_dtypes_n_unique_idx
+    xs = [np.asarray(x, dtype=dt) for x, dt in zip(xs, input_dtypes)]
+    helpers.test_frontend_function(
+        input_dtypes=input_dtypes,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="torch",
+        fn_name="reshape",
+        tensors=xs,
```

Comment:
please update parameters here to match parameters in `torch.reshape`, with variables mapped to their respective arguments

Reply:
question: 
the args of helpers.test_frontend_function() should be the expected args of the backend functions, in this case, since torch.reshape only asks for 2 args, input: Tensor, shape: Tuple(int), we would only include these two args for helpers.test_frontend_function(), correct?